ePBFT/src/pbft/libbyz/Replica.cpp

```diff
@@ -305,6 +300,7 @@ bool Replica::apply_ledger_data(const std::vector<uint8_t>& data)
     }
     else
     {
```

Comment:
Perhaps add a `LOG_FAIL_FMT()` if the execution fails?

Reply:
It isn't failing exactly and I think a log every time this returns false is too much. It can just indicate that the entries I have received are ahead of my own because I am lagging, so I need to request newer entries. I can `LOG_DEBUG` though.